python/lsst/meas/base/references.py

```diff
@@ -244,9 +163,8 @@ class CoaddSrcReferencesTask(BaseReferencesTask):
     ----------
     schema : `lsst.afw.table.Schema`, optional
         The schema of the detection catalogs used as input to this one.
-    butler : `lsst.daf.persistence.butler.Butler`, optional
-        A Butler used to read the input schema from disk. Required if
-        ``schema`` is `None`.
+    butler : `None`
+        Compatibility parameter. Should always be `None`.
```

Comment:
Class description still says: "The dataset is then read directly from disk using the Butler."

Are you keeping this class around just because `MultiBandReferencesTask` inherits it?  It doesn't seem to do anything anymore!

Reply:
Ok. I'm removing the entire file.